src/settings/nm-settings.c

```diff
@@ -535,11 +535,13 @@ get_plugin (NMSettings *self, NMSettingsPluginCapabilities capability)
 
 	/* Do any of the plugins support the given capability? */
 	for (iter = priv->plugins; iter; iter = iter->next) {
-		NMSettingsPluginCapabilities caps = NM_SETTINGS_PLUGIN_CAP_NONE;
+		NMSettingsPlugin *plugin = NM_SETTINGS_PLUGIN (iter->data);
+
+		if (has_add_connection != TRUE)
```

Comment:
can we avoid comparing boolean values `!=` and `==` with `TRUE|FALSE`?

Also, at several places the branch compares pointers against NULL, which I think should be avoided. But I care less about pointer comparisons than about booleans. On the contrary, I think integers should be compared explicitly against `0`, not just `if (value)`. Yes, we are not very consistent about all this.